File: JAVA/AutoProject/Tesla.java
import java.util.Random;

public class Tesla implements Auto {
    private double sebesseg;
    private static Random random = new Random();

    public Tesla() {
        sebesseg = random.nextInt(61);
    }

    @Override
    public void gyorsul() {
        sebesseg += random.nextInt(11) + 5;
    }

    @Override
    public double getSebesseg() {
        return sebesseg;
    }
}

File: C++/AutoProject/Tesla.cpp
#include "Tesla.h"

Tesla::Tesla() {
    std::srand(std::time(0));
    sebesseg = std::rand() % 61;
}

void Tesla::gyorsul() {
    sebesseg += std::rand() % 11 + 5;
}

double Tesla::getSebesseg() const {
    return sebesseg;
}